force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -57,8 +66,17 @@ public with sharing class FlowControlLoopsCollections {
     public static Integer findMax(Integer num1, Integer num2) {
         // Initialize largestNum as null
         Integer largestNum = null;
+        if (num1 == num2) {
+            return num1;
+        } else if (num1 < num2) {
+            return num2;
+        } else if (num1 > num2) {
+            return num1;
+        } else {
+            return largestNum;
+        }
```

Comment:
Try using the Math class https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_math.htm#apex_System_Math_max